lib/game/game_render_box.dart

```diff
@@ -79,4 +79,7 @@ class GameRenderBox extends RenderBox with WidgetsBindingObserver {
   void didChangeAppLifecycleState(AppLifecycleState state) {
     game.lifecycleStateChange(state);
   }
+
+  @override
+  Size computeDryLayout(BoxConstraints constraints) => constraints.biggest;
```

Comment:
Didn't we need some annotation to suppress the warning for overriding something that doesn't exist on stable?

Reply:
I would think leaving out the `@override` annotation would mean analysis would pass on flutter stable, and give a warning on beta, which I think would be desirable.

Reply:
or you can add the comment `  // ignore: override_on_non_overriding_member`

Reply:
That crossed my mind, but Renan reminded us that pre releases (which is the case of our RCs) aren't analyzed by pub and I though on leaving this correct since there is a (hopefully big) chance that the change will already be on stable when we release the final V1. What do you think @spydon @christopherfujino